packages/core/src/internal/fetchOptions.ts

```diff
@@ -49,7 +49,9 @@ export const makeFetchURL = (
   }
 
   if (body.query) {
-    search.push('query=' + encodeURIComponent(body.query));
+    search.push(
+      'query=' + encodeURIComponent(body.query.replace(/[\s,]+/g, ' ').trim())
```

Comment:
How come we replace with a space? Also is it risky to remove commas?

Reply:
The Graphql Query spec only needs the text to be seperated so we go for the smallest amount of space possible (commas aren't enforced)